### ISSUE TITLE
FluffyChat web's design is too isolated, browser accessibility becomes void

### ISSUE THREAD

**Issue by GrantedKelp90** [krille-chan/fluffychat#3120]
### Bug Description

I cannot style or modify FluffyChat's appearance using browser extensions and accessibility tools, making it harder for me to navigate the site. I use a custom font across my browser and applications, but FluffyChat is exempt to this and becomes immune to any changes.

Extension tools like Dark Reader, FontChanger, and ScrollAnywhere are all completely broken.

### Steps to Reproduce

I am loading FluffyChat on Librewolf, using simple browser tools. In this situation, I cannot make the site more accessible for my eyes.

### Expected Behavior

All the tools I described should function correctly, as they do with all other websites I regularly visit.

### App Version

2.6.0

### Platform

Web (Firefox)

### Additional Platform Information

N/A

### Additional Context

No AI was used in the making of this report. I say this as I've been told that my writing style is similar.

I will be creating a separate feature request to add custom font support, and another for proper Material integration on Android.

**Comment by krille-chan** [comment]
I'm very sorry but there is no way to solve this. FluffyChat is using the Flutter framework to reuse the same code for web, mobile and desktop. While this is really great to create apps with (honestly) native performance, it has this drawback on web 😢

As there is nothing I can do, I would close this issue.
